ontology/yaml/resources/HVAC/entity_types/DC.yaml

```diff
@@ -24,3 +24,13 @@ DC_SS_SWTC:
   - DC
   - SS
   - SWTC
+
+DC_SS_SWTC_CDWISOVM:
+  description: "Typical dry-cooler (closed loop sensible cooler)."
+  is_canonical: true
+  implements:
+  - DC
+  - SS
+  - SWTC
+  - CDWISOVM
```

Comment:
Instead of using `CDWISOVM` I suggest using a more-general isolation valve type (e.g., `SWISOVM` or `RWISOVM`)

The "condensing" description on the valve is superfluous since this is a dry cooler with only one type of water.

Reply:
Hi Claire, we are waiting for confirmation regards the isolation vales position to return on that.

Reply:
Claire, as suggested the isolation valve type has been changed to RWISOVM as it is a return valve according to contractor.